### ISSUE TITLE
how to create slide panel on slide panel (multiple level)?

### ISSUE THREAD

**Issue by thearabbit** [rjobanp/meteor-slide-panel#2]
for example i have `slide 1`, and in it have `btnslide2`.
when i click to show `slide 2`, but `slide 1` is closed.
I would to create slide panel on slide panel (multiple level slide).
how?

**Comment by rjobanp** [comment]
Hi @yuomtheara . Currently this package only supports one visible slide panel at a time. The slide panel is only one main DOM element and the template you choose to show is dynamically inserted into that same element each time.

Let's say you had `slide 1` open and you then opened `slide 2`. You can then show `slide 1` automatically after `slide 2` is closed to simulate having two panels on top of each other:
Right after calling `slidePanel.showPanel('slide 2')` just pass a function to `slidePanel.closePanel` which calls `slidePanel.showPanel('slide 1')`.

**Comment by thearabbit** [comment]
Thanks for your reply, so it mean that when the  `slide 2` closed the `slide 1` is opened (it will reset data on `slide 1`). i don't want to do that, because i want to change any data on `slide 1` atfer `slide 2` closed.

**Comment by rjobanp** [comment]
Yes it will reset the state of the template when the panel is closed and
opened by default. One thing you could do is save the state of the `slide
1` into a local or session variable when it closes, and then pass that as
the data value for the template when you reopen it again to set the state
as it was.

**Comment by thearabbit** [comment]
thanks, but i would like to request the multiple level option.
